bamt/utils/composite_utils/MLUtils.py

```diff
@@ -0,0 +1,97 @@
+from catboost import CatBoostClassifier, CatBoostRegressor
+from lightgbm.sklearn import LGBMClassifier, LGBMRegressor
+from sklearn.cluster import KMeans
+from sklearn.ensemble import (
+    AdaBoostRegressor,
+    ExtraTreesRegressor,
+    GradientBoostingRegressor,
+    RandomForestClassifier,
+    RandomForestRegressor,
+)
+from sklearn.linear_model import (
+    Lasso,
+    LinearRegression,
+    LogisticRegression,
+    Ridge,
+    SGDRegressor,
+)
+from sklearn.naive_bayes import BernoulliNB, MultinomialNB
+from sklearn.neural_network import MLPClassifier
+from sklearn.svm import LinearSVR
+from sklearn.tree import DecisionTreeClassifier, DecisionTreeRegressor
+from xgboost import XGBClassifier, XGBRegressor
+import json
+from .CompositeModel import CompositeNode
+from random import choice
+
+
+class MlModels:
+    def __init__(self):
+        self.operations_by_types = {
+            "xgbreg": "XGBRegressor",
+            "adareg": "AdaBoostRegressor",
+            "gbr": "GradientBoostingRegressor",
+            "dtreg": "DecisionTreeRegressor",
+            "treg": "ExtraTreesRegressor",
+            "rfr": "RandomForestRegressor",
+            "linear": "LinearRegression",
+            "ridge": "Ridge",
+            "lasso": "Lasso",
+            "sgdr": "SGDRegressor",
+            "lgbmreg": "LGBMRegressor",
+            "catboostreg": "CatBoostRegressor",
+            "xgboost": "XGBClassifier",
+            "logit": "LogisticRegression",
+            "bernb": "BernoulliNB",
+            "multinb": "MultinomialNB",
+            "dt": "DecisionTreeClassifier",
+            "rf": "RandomForestClassifier",
+            "mlp": "MLPClassifier",
+            "lgbm": "LGBMClassifier",
```

Comment:
Will we leave `lgbm`?

Reply:
Yep, but it's debateable. I'll comment it out for now